src/extension.ts

```diff
@@ -48,9 +59,10 @@ function checkFile(doc: vscode.TextDocument) {
     let shebang = doc.lineAt(0);
 
     /*
-     * Do nothing if the first line is not a shebang line.
+     * Do nothing if the first line is not a shebang-like line.
      */
-    if (!shebang.text.startsWith("#!")) {
+    const match = acceptedPrefixes.find(prefix => shebang.text.startsWith(prefix));
+    if (!match) {
```

Comment:
I think `Array.some` might be a better fit here?

```suggestion
    const match = acceptedPrefixes.some(prefix => shebang.text.startsWith(prefix));
    if (!match) {
```